test/Solnet.Programs.Test/Utilities/DeserializationUtilitiesTest.cs

```diff
@@ -271,7 +271,7 @@ public void TestReadRustStringException()
         public void TestReadRustString()
         {
             const string expected = "this is a test string"; // 21 chars
-            const int expectedLength = 25; // 21 chars + sizeof(uint) which is the encoding length padding
+            const int expectedLength = 29; // 21 chars + sizeof(u64) which is the encoding length padding

```

Comment:
make it explicit `expectedLength = expected.Length + sizeof(ulong)` and there's no need for the comment.

But add a comment stating that it doesn't work for all characters due to utf-8 encoding.